src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEModuleSymbol.cs

```diff
@@ -702,5 +708,43 @@ internal IEnumerable<NamedTypeSymbol> GetForwardedTypes()
         }
 
         public override ModuleMetadata GetMetadata() => _module.GetNonDisposableMetadata();
+
+        internal bool ShouldDecodeNullableAttributes(Symbol symbol)
+        {
+            Debug.Assert(!(symbol is null));
+            Debug.Assert(symbol.IsDefinition);
+            Debug.Assert((object)symbol.ContainingModule == this);
+
+            if (_lazyNullableMemberMetadata == NullableMemberMetadata.Unknown)
+            {
+                _lazyNullableMemberMetadata = GetAttributes().IndexOfAttribute(this, AttributeDescription.NullablePublicOnlyAttribute) >= 0 ?
```

Comment:
>GetAttributes() [](start = 46, length = 15)

Should use well-know attributes infrastructure instead (special method on PEModule, etc.)